load_fixtures.yml

```diff
@@ -12,9 +12,11 @@
 
   tasks:
 
-  - debug: msg="Project name {{ project_name }} - domain {{ domain_name }}"
+  - name: Print project informations
```

Comment:
the word "information" does not have a plural in english. Aplies to other places where you used it.